recipe/meta.yaml

```
@@ -1,45 +1,54 @@
-{% set name = "<PLEASE ADD PKG NAME>" %}
-{% set version = "<PLEASE ADD PKG VERSION>" %}
+{% set name = "conda-anaconda-telemetry" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "01044e9d7cc51eab3fa70e0ffa0b7638e20c90ce00e1b346e97d94288d1111fc" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: <sha256>
+  url: https://github.com/anaconda/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  skip: True  # [py<38]
+  # keeping as noarch so that the plugin may be used with architectures which are no longer actively maintained (e.g. osx-64)
+  noarch: python
+  number: 1
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
+  build:
+    - git # [not win]
   host:
-    - python
+    - python 3.9
     - pip
+    - hatchling >=1.12.2
+    - hatch-vcs >=0.2.0
   run:
-    - python
+    - python >=3.9
+    - conda >=24.11
 
 test:
   imports:
-    - {{ name.replace('-', '.') }}
+    - conda_anaconda_telemetry
+    - conda_anaconda_telemetry.hooks
   requires:
     - pip
   commands:
     - pip check
+    - conda --version
```

```suggestion
    - python -m conda --version
```
To make sure that we are testing the right thing?

---

Good catch!